packages/react-component-library/src/components/TabSet/TabItem.tsx

```diff
@@ -37,22 +37,23 @@ export const TabItem = forwardRef<HTMLLIElement, TabItemProps>(
         $isFullWidth={isFullWidth}
         $isScrollable={isScrollable}
         ref={ref}
-        role="tab"
-        aria-controls={tabId}
-        aria-selected={!!isActive}
-        tabIndex={!isActive ? -1 : 0}
+        role="presentation"
         data-testid="tab-set-tab"
-        aria-label={children.toString()}
-        onKeyDown={onKeyDown}
       >
         <StyledTab
           $isActive={isActive}
           $isScrollable={isScrollable}
+          id={`tab-button-${tabId}`}
```

Comment:
The error here was:

https://dequeuniversity.com/rules/axe/4.4/nested-interactive

This was due to the `tab` role containing a `button` role. I simply rejigged things so that the `<button>` is now the `tab` (following the same pattern as the GOV.UK Design System).

Also removed `aria-label` as a button gets an accessible name automatically, and `children.toString()` was `[object Object]` in some cases.